### ISSUE TITLE
Token logos are too large/heavy

### ISSUE THREAD

**Issue by katspaugh** [gnosis/safe-client-gateway#763]
**Describe the bug**
I did a spot check on different coin icons we are receiving from the gateway.

https://safe-transaction-assets.gnosis-safe.io/chains/4/currency_logo.png – 3258x3258 px, 161 Kb
https://safe-transaction-assets.gnosis-safe.io/chains/43114/currency_logo.png – 1500x1500, 41 Kb

I suppose many other icons are also unnecessarily big. This is very bad for the page loading speed. These icons weigh more than the page itself, including the JS.

## Frontend requirements
* Token logos should be sufficiently big, so that we can downscale them to different sizes in different components.
* But not too big. 512x512 would be good enough for most situations, including hi-res (aka Retina) screens.
* Images should be optimized, e.g. via OptiPng. This will already half the current sizes. Try it on optipng.com and there's a CLI util of the same name.

## To summarize
Please do the following:
* Resize all the existing icons to 512x512 px
* Run them through optipng
* Set up a pipeline to do the same for new icons

**Comment by Uxio0** [comment]
I will take care of resizing the logos @fmrsabino

**Comment by Uxio0** [comment]
@katspaugh Logos were updated to optimized png `512x512` when possible, both in staging and mainnet. I'm moving the issue to the `config-service` for further analysis of `Set up a pipeline to do the same for new icons`
